specification/gp-connect-prescriptions-management-fhir.yaml

```diff
@@ -963,4 +966,342 @@ components:
         type: string
         pattern: '^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$'
         example: 60E0B220-8136-4CA5-AE46-1D97EF59D068
-     
+  schemas:
+    TaskForRequest:
+      type: object
+      description: |
+        Schema for a task resource
+      properties:
+        resourceType:
+          type: string
+          description: FHIR resource type.
+          example: Task
+        id: 
+          type: string
+          description: identifier of the resource itself.
+          example: 7c747c6e-15c8-4b2f-8aa0-441f2ef703d6
+        identifier:
+          type: array
+          items:
+            type: object
+            properties:
+              system:
+                type: string
+                description: codesystem URL for identifiers
+                example: https://tools.ietf.org/html/rfc4122
+              value:
+                type: string
+                description: Business identifier of the resource.
+                example: e7c83c02-d275-4f22-b034-69b992bbaf5f
+        status: 
+          type: string
+          description: status of the task itself, descriptions of each status can be found [here.](https://simplifier.net/guide/GP-Connect-Prescription-Management-FHIR-PFS/Home/FHIR-Assets/All-Assets/Profiles/UKCore-Task?version=current)
+          enum:
+            - requested
+            - accepted
+            - rejected
+            - cancelled
+            - in-progress
+            - completed
+            - failed
+        statusReason:
+          type: object
+          description: To be completed upon rejections to feedback to the patient.
+          properties:
+            text:
+              type: string
+              example: No longer suitable please contact GP
+        intent: 
+          type: string
+          description: Needed for FHIR compliance reasons, the value should always be set to order.
+          example: order
+        code:
+          type: array
+          description: Codeable concept to state the purpose of the task.
+          items:
+            type: object
+            properties:
+              coding:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    system:
+                      type: string
+                      example: http://snomed.info/sct
+                    code: 
+                      type: string
+                      example: 103742009
+                    display: 
+                      type: string
+                      example: Renewal of prescription
+        focus:
+          type: string
+          description: Must have a reference of a previous medication request.
+          example: MedicationRequest/b269d1d7-1acf-47bb-8b3c-e38b583d9a07
```

Comment:
It might be worth specifying a [pattern](https://swagger.io/docs/specification/data-models/data-types/#pattern) to help people understand the need for `MedicationRequest/` to begin the value - if that is the case.